### PR TITLE
Qualify package names

### PR DESCRIPTION
**Pull request by r-ash**
I'm trying to add github actions to naomi (instead of using buildkite)

That is raising an error because a call in this package can't find `unzip` function see https://github.com/mrc-ide/naomi/actions/runs/8283097208/job/22665482224?pr=433#step:8:7663

or with `setNames` function https://github.com/mrc-ide/naomi/actions/runs/8283701894/job/22667510478?pr=433#step:7:7661

This PR qualifies all package names which were being raised in an R CMD note. Regenerated the docs, added some missing imports into the DESCRIPTION. I have removed the old `.travis.yml` whilst I was here as that is no longer used.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -11,6 +11,8 @@ Imports:
     epp (>= 0.4.1),
     fastmatch (>= 1.1),
     mvtnorm,
+    plyr,
```
Comment (jeffeaton) [inline DESCRIPTION:14]: Where is `plyr` used? I don't have recollection of using that.

**Review by jeffeaton — CHANGES_REQUESTED** [review]
Thanks, wow you touched some really niche code here. Thank you for cleaning this up. 

One question about what `plyr` dependency is needed for. Not disputing, but I don't have recollection of it.

Shall we bump the version number and NEWS?

File: DESCRIPTION (hunk 1 above)
Comment (r-ash) [reply DESCRIPTION:14]: Looks like this is only used in 1 place actually, https://github.com/mrc-ide/eppasm/blob/master/R/read-spectrum-pop1.R#L43

**Comment by r-ash** [comment]
> Shall we bump the version number and NEWS?

Good shout, I've updated that

File: DESCRIPTION (hunk 1 above)
Comment (jeffeaton) [reply DESCRIPTION:14]: hmmm, not really needed there. but not worth the effort to unwind it!

**Review by jeffeaton — APPROVED** [review]
lgtm

**Pull request merged**